assets/js/googlesitekit/modules/datastore/modules.js

```diff
@@ -20,19 +20,33 @@
  * External dependencies
  */
 import invariant from 'invariant';
+import { keyBy, sortBy } from 'lodash';
 
 /**
  * Internal dependencies
  */
 import API from 'googlesitekit-api';
 import Data from 'googlesitekit-data';
 import { STORE_NAME } from './constants';
+import { STORE_NAME as CORE_SITE_STORE_NAME } from '../../../googlesitekit/datastore/site';
```

Comment:
We've usually imported these from the constants file (reduces possibility of circular imports), and in other files we've imported it as `CORE_SITE`:

```suggestion
import { STORE_NAME as CORE_SITE } from '../../../googlesitekit/datastore/site/constants';
```